CONTRIBUTING.md

```diff
@@ -122,3 +122,9 @@ async def test_haruna_cute(test_cli):
     res = await test_cli.post('/replace/with/lesson/frontend/url/', data=json.dumps({ Data dictionary you want to test }))
     ...
 ```
+
+## Testing Your Lessons
+Once you made tests for your lesson, be sure to test it locally before making a pull request!
+You can easily do this by running `python -m pytest`. Pytest will alert you which of your tests fail so you can easily fix them! If you do not catch these locally, the repository has Travis integration to make sure your builds pass or fail, so be sure to check your pull request after you make it to see if you succeed!
```

Comment:
Maybe add more command such a `python -m pytest /tests`  can specific a test file to test.